pyogrio/geopandas.py

```diff
@@ -67,6 +68,9 @@ def read_dataframe(
         specific (e.g. typically 0 for Shapefile and 1 for GeoPackage, but can
         still depend on the specific file). The performance of reading a large
         number of features usings FIDs is also driver specific.
+    set_fid_index : bool, optional (default: False)
+        If True, will return use the FIDs of the features that were read as the
```

Comment:
```suggestion
        If True, will use the FIDs of the features that were read as the
```
Not sure which of those verbs were intended to be there but likely not both.